tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.ts

```diff
@@ -162,6 +162,17 @@ export class LineChartAxisComponent {
     return this.scale.ticks(this.axisExtent, 2);
   }
 
+  shouldShowMinorTick(tick: number, axis: 'x' | 'y'): boolean {
+    // Hide the tick text when it is too close to the left/right edge.
+    const fractionOfFullExtent =
+      (tick - this.axisExtent[0]) / (this.axisExtent[1] - this.axisExtent[0]);
+    const margin = axis === 'x' ? 0.075 : 0.03;
+    if (fractionOfFullExtent < margin || fractionOfFullExtent > 1 - margin) {
+      return false;
+    }
+    return true;
+  }
+
```

Comment:
This feels too brittle. 1. There is no guarantee that the text would not bleed out even with these margins and (2) constants defined up feels too magical. Can we just start without this? If you find it important, use canvas to measure and properly quell the axis? 

Regardless, we need to update the test.

Reply:
Sure, I can work on a proper fix + test, but I definitely prefer having this noticeable regression fixed before enabling the fastChart flag by default.

Regarding the approach, measurement sounds good to me.  To make sure we're on the same page, it sounds like the process will be:
- Get all the tick positions
- Measure the first and last tick text
- If the last tick goes beyond the right edge, hide it (vice versa for first tick/left edge)

Also, I'm not sure how to turn a non-draft PR back into a draft, so I'll just update the header title.

Reply:
Regression is not a functional but cosmetic. I think we should not be blocking something based on this mere minor "inconvenience".